assets/wizards/googleAdManager/index.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * Google Ad Manager Wizard.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, render, Fragment } from '@wordpress/element';
```

Comment:
`Fragment` doesn't seem to be used in this component:

```suggestion
import { Component, render } from '@wordpress/element';
```